src/Stdin.vue

```diff
@@ -82,7 +86,8 @@ export default {
     command: '',
     // Determinate if input is disabled
     isDisabled: false,
-    placeholder: ''
+    placeholder: '',
+    locPrompt: ''
```

Comment:
Please rename it to `localPrompt`. I think it's easier to understand for other contributors.